wolfssl-3.9.6/README.md

```diff
@@ -0,0 +1,139 @@
+Finds a SEGV and three heap overflow errors in `src/ssl.c`
+in [wolfSSL v.3.9.6](https://www.wolfssl.com/wolfSSL/Home.html).
```

Comment:
I believe all of these were fixed in the same commit because they were also reported all together. In that case feel free to close the PR!